src/main/python/cfn_sphere/cli.py

```diff
@@ -156,9 +158,18 @@ def sync(config, parameter, debug, confirm, yes, context, dry_run):
             get_first_account_alias_or_account_id()), abort=True)
 
     try:
+        cached_stack_data = {}
+        if session:
+            with open(session) as session_data:
```

Comment:
This bit isn't going to work as we won't have a session file the first time round.